wallet/wallet-controller/src/synced_controller.rs

```diff
@@ -569,7 +569,7 @@ where
         &mut self,
         destination_address: Destination,
         from_addresses: BTreeSet<Destination>,
```

Comment:
The fact that empty `from_addresses` now means "all addresses" is super unintuitive and must be documented. And in multiple places too:
*) The doc comment for this function must mention this.
*) Well, probably every `sweep_addresses` function should have a doc comment mentioning this (if it doesn't have an explicit "all" parameter).
*) Need a comment where WalletCommand::SweepFromAddress is handled, because it's supper confusing - you check "all" and then forget about it.

But IMO it's better to just pass the "all" flag explicitly everywhere (and require that if `all` is `true`, then `from_addresses` must be empty).